src/packages/solid/components/Badge/Badge.tsx

```diff
@@ -37,12 +37,12 @@ export interface BadgeProps extends NodeProps {
    * Object containing all properties supported in the [Icon component](?path=/docs/components-icon--icon)
    */
   icon?: Partial<IconProps>;
-}
+};
 
 const padding = styles.padding;
 const Badge: Component<BadgeProps> = (props: BadgeProps) => {
   return (
-    <node use:withPadding={padding} style={styles.Container} {...props}>
+    <node use:withPadding={padding} style={styles.container} {...props}>
```

Comment:
instead of changing this to lowercase, let's update the badges style file to capitalize Container